docs/QUICKSTART.md

```diff
@@ -0,0 +1,171 @@
+# Hellow Quickstart
+
+This guide is partially complete. If you are following this guide, please
+contribute information on any places where you've gotten stuck and how
+to work through that issue.
+
+This document assumes some base level understanding of the moving AWS parts
+involved, as well as go. This also assumes `hallow-cli` has been installed,
+and can be run on the command line, as well as the ability for your local
+computer to talk to the AWS API.
+
+## Push Hallow deployment zip to S3
+
+Pull the latest Hallow zip from GitHub actions, or build your own zip using
+`make`, and put that zip in an S3 bucket. When running Terraform, you will
+need the bucket name, as well as the name of the file (should be `hallow.zip`).
+
+## Create Hallow in your AWS Environment
+
+First, we need to set up Hallow to run in your AWS environment. This will ask
+you where the bucket and file are, as well as which region you're in.
+
+```
+cd terraform
+terraform apply
```

Comment:
Pretty sure this doesn't work, I'm almost positive you need to instantiate the module yourself.

Reply:
@alex I was able to deploy it just using `terraform apply` and filling out the questions :laughing: - want to update this bit, though?

Reply:
Huh. Ok! Maybe it should say something like "for production deployments you'll probably want to instantiate this as a module"?

Reply:
@alex I can remove this if you want to put best practice in instead!

Reply:
(I'm only a few months into using TF, all my usage is entirely by accident)

Reply:
Seems ok for a quickstart guide, I genuinely did not expect it to work. I think it'd be good to also have a quick reference to the fact that for real life you'll probably want something a tad fancier.

Reply:
@alex note added!